### PR TITLE
feat: add Format, Statement of Responsibility, References

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to
+ [Statement of Responsibility APPS-1797](https://jira.library.ucla.edu/browse/APPS-1797)
+ [Format APPS-1816](https://jira.library.ucla.edu/browse/APPS-1816)
+ [AdminNote APPS-1817](https://jira.library.ucla.edu/browse/APPS-1817)
+ [References APPS-1818](https://jira.library.ucla.edu/browse/APPS-1818)

---

#### AdminNote, References, & Format
https://ursus-test.library.ucla.edu/catalog/ark:/21198/z13f64qn
Screen Shot 2023-03-16 at 5 28 16 PM

#### Statement of Responsibility
https://ursus-test.library.ucla.edu/catalog/ark:/21198/z1pg5rpw
Screen Shot 2023-03-16 at 5 32 29 PM

OAI for statement of responsibility, format & references
+ https://ursus-test.library.ucla.edu/catalog/oai?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai:library.ucla.edu:ark:/21198/z13f64qn
+ https://ursus-test.library.ucla.edu/catalog/oai?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai:library.ucla.edu:ark:/21198/z1pg5rpw

### CODE REVIEW

File: app/services/search_field_service.rb [hunk 1]
```diff
@@ -61,7 +61,11 @@ class SearchFieldService
     'host_tesim',
     'musician_tesim',
     'printer_tesim',
-    'researcher_tesim'
+    'researcher_tesim',
+    'format_book_tesim',
+    'note_admin_tesim',
```
Comment (pghorpade) [inline app/services/search_field_service.rb:66]: remove note_admin_tesim, as per the ticket, this should not be searchable in Ursus

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -219,28 +219,28 @@ class CatalogController < ApplicationController
     config.add_show_field 'provenance_tesim', label: 'Provenance'
     config.add_show_field 'colophon_tesim', label: 'Colophon'
     config.add_show_field 'note_tesim', label: 'Note'
+    config.add_show_field 'note_admin_tesim', label: 'AdminNote'
```
Comment (pghorpade) [inline app/controllers/catalog_controller.rb:222]: AdminNote should be Admin Note

File: config/locales/blacklight.en.yml [hunk 3]
```diff
@@ -102,8 +102,10 @@ en:
           alternative_title_tesim: 'Alternative title'
           architect_tesim: 'Architect'
           ark_ssi: 'ARK'
+          admin_note_tesim: 'AdminNote'
```
Comment (pghorpade) [inline config/locales/blacklight.en.yml:105]: should there be a space between Admin Note,

File: config/metadata/note_metadata.yml [hunk 4]
```diff
@@ -6,6 +6,9 @@ colophon_tesim: 'Colophon'
 provenance_tesim: 'Provenance'
 note_tesim: 'Note'
 # toc_tesim: 'Table of Contents' (render_table_of_contents_key / value)
+note_admin_tesim: 'AdminNote'
```
Comment (pghorpade) [inline config/metadata/note_metadata.yml:9]: should it be Admon Note, did you verify the order with Dawn?

File: spec/services/search_field_service_spec.rb [hunk 5]
```diff
@@ -5,9 +5,6 @@
...
...itect_tesim', 'author_tesim', 'binding_note_tesim', 'caption_tesim', 'collation_tesim', 'colophon_tesim', 'composer_tesim', 'condition_note_ssi', 'contents_note_tesim', 'contributor_tesim', 'creator_tesim', 'date_created_tesim', 'description_tesim', 'explicit_tesim', 'features_tesim', 'foliation_tesim', 'genre_tesim', 'human_readable_language_tesim', 'identifier_tesim', 'illuminator_tesim', 'illustrations_note_tesim', 'inscription_tesim', 'incipit_tesim', 'local_identifier_ssm', 'location_tesim', 'lyricist_tesim', 'medium_tesim', 'named_subject_tesim', 'page_layout_ssim', 'opac_url_ssi', 'page_layout_ssim', 'photographer_tesim', 'place_of_origin_tesim', 'provenance_tesim', 'publisher_tesim', 'scribe_tesim', 'script_tesim', 'series_tesim', 'combined_subject_ssim', 'summary_tesim', 'support_tesim', 'title_tesim', 'toc_tesim', 'uniform_title_tesim', 'hand_note_tesim', 'writing_system_tesim', 'uniform_title_tesim', 'shelfmark_ssi', 'descriptive_title_tesim', 'delivery_tesim', 'other_versions_tesim', 'repository_tesim', 'host_tesim', 'musician_tesim', 'printer_tesim', 'researcher_tesim', 'format_book_tesim', 'note_admin_tesim', 'resp_statement_tesim', 'citation_source_tesim'].join(' '))
```
Comment (pghorpade) [inline spec/services/search_field_service_spec.rb:8]: remove note_admin_tesim from here too

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: config/metadata/note_metadata.yml (hunk 4 above)
Comment (jendiamond) [reply config/metadata/note_metadata.yml:9]: yes she said it doesn't matter

**Review by pghorpade — APPROVED** [review]
👍

File: app/services/search_field_service.rb (hunk 1 above)
Comment (jendiamond) [reply app/services/search_field_service.rb]: removed

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**